awscli/examples/application-signals/list-service-dependents.rst

```diff
@@ -0,0 +1,36 @@
+**To return the list of dependents that invoked the specified service during the provided time range**
+
+The following ``list-service-dependents`` example returns the list of dependents that invoked the specified service during the provided time range. ::
+
+    aws application-signals list-service-dependents \
+        -—start-time 1732021200 \
+        --end-time 1732107600 \
+        -—key-attributes Environment=generic:default,Name=PetSite,Type=Service
```

Comment:
Lines 6 and 8 ended up with a non-ASCII dash, can you make sure they're only `--`?